vllm/entrypoints/chat_utils.py

```diff
@@ -92,6 +93,16 @@ class CustomChatCompletionContentSimpleAudioParam(TypedDict, total=False):
     audio_url: Required[str]
 
 
+class CustomChatCompletionContentInputAudioParam(TypedDict, total=False):
+    # Same as InputAudio type from https://github.com/openai/openai-python/blob/main/src/openai/types/chat/chat_completion_content_part_input_audio_param.py
+    data: Required[str]
+    """Base64 encoded audio data."""
+
+    format: Required[Literal["wav", "mp3"]]
+    """The format of the encoded audio data. 
+    Currently supports "wav" and "mp3"."""
+
+
```

Comment:
From my understanding, we should only accept `ChatCompletionContentPartAudioParam`, but not `CustomChatCompletionContentInputAudioParam` directly.

Reply:
Actually this `CustomChatCompletionContentInputAudioParam` is not really used in the code. I will remove it. For the new API, parameters will be casted to `ChatCompletionContentPartInputAudioParam`, which is directly ported from OAI